osbs/build/build_request.py

```diff
@@ -649,10 +664,63 @@ def render_import_image(self, use_auth=None):
                 self.dj.dock_json_set_arg('postbuild_plugins', 'import_image',
                                           'insecure_registry', True)
 
+    def render_customizations(self):
+        """
+        Customize prod_inner for site specific customizations
+        """
+
+        try:
+            for plugin_dict in self.customize_conf['disable_plugins']:
```

Comment:
It might be more "pythonic" to remove one-level of nested code by replacing the outer try/except with `self.customize_conf.get('disable_plugins', [])`

This will return an empty list of customize_conf does not provide any 'disable_plugins'.

same for the `self.customize_conf['enable_plugins']` statement below.